components/cam/src/physics/clubb/advance_clubb_core_module.F90

```diff
@@ -105,8 +107,8 @@ module advance_clubb_core_module
                                        ! predictive fields
 
   ! Select the placement of the call to CLUBB's PDF.
-  integer, parameter :: &
-    ipdf_call_placement = ipdf_pre_advance_fields
+  !integer, parameter :: &
```

Comment:
This commented out code can be completely removed.

Reply:
Removed. Thanks!